src/App.js

```diff
@@ -9,34 +8,39 @@ class App extends Component {
     super();
     this.state = {
       movies: [],
-      singleMovie: movieData.singleMovie,
+      singleMovie: {},
```

Comment:
We got this updating the empty object.

Reply:
An empty object totally works! You could also reach for Null if you wanted to!